packages/block-library/src/navigation/index.php

```diff
@@ -212,16 +228,22 @@ function block_core_navigation_build_html( $attributes, $block, $colors, $font_s
 			$css_classes .= ' ' . $class_name;
 		};
 
+		$item_url = get_navigation_link_url( $block['attrs'] );
+		$item_tag = $item_url ? 'a' : 'div';
```

Comment:
Why isn't this a `span` instead?

Reply:
Yep good question :) a span is more fit as an inline non-link element.